llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -417,16 +425,21 @@ static void collectSymbolsLists(
   }
 }
 
+struct ResultModule {
+  StringRef KernelModuleName;
+  std::unique_ptr<Module> ModulePtr;
+};
+
 // Input parameter KernelModuleMap is a map containing groups of kernels with
```

Comment:
Definitely **Not** asking to do such change in scope of this PR (to make it cleaner), but it seems to me that this comment (and `KernelModuleMap` name as well) has been outdated for some time now, because this map contains more than just kernels now